torch/fx/graph_module.py

```diff
@@ -310,6 +330,14 @@ def wrapped_call(self, *args, **kwargs):
                 sys.excepthook = old_excepthook
         cls.__call__ = wrapped_call
 
+    def __reduce_package__(self, exporter: PackageExporter):
+        generated_module_name = f'fx-generated._{exporter.get_unique_id()}'
```

Comment:
is `fx-generated` a valid atom in a qualname? At first sight I thought this would be parsed as operator -

Reply:
it is, but it is also not usable in an import statement, i.e. `import fx-generated._0` is a syntax error. I actually think this is a desirable property, because it virtually guarantees we will not shadow a user-defined module. But I could change to whatever.